x-pack/plugins/serverless_search/public/plugin.ts

```diff
@@ -41,8 +42,13 @@ export class ServerlessSearchPlugin
         const [coreStart, services] = await core.getStartServices();
         const { security } = services;
         docLinks.setDocLinks(coreStart.docLinks.links);
-
-        const userProfile = await security.userProfiles.getCurrent();
+        let userProfile: UserProfileUserInfo | undefined;
```

Comment:
I'm good with this as a quick fix, but we should likely move this to be async outside of the setup flow. Maybe to a context or something or fetch on the page.